src/engine/state.ts

```diff
@@ -1058,6 +1067,20 @@ if (!db) {
     ctx.app.dufflepudUrl = getSetting("dufflepud_url")
   })
 
+  ctx.net.pool.on("init", (connection: Connection) => {
+    connection.on(ConnectionEvent.Receive, function (cxn, [verb, ...args]) {
+      if (verb == "EVENT" || verb == "EOSE" || verb == "NEG-MSG") return
```

Comment:
Should be `EVENT|EOSE|CLOSED`

Reply:
Also, triple equals (humor me)

Reply:
> Let's actually add CLOSED back in (my mistake)
Those are the events excluded. If I replace NEG-MSG with CLOSED we won't see CLOSED message.

Reply:
Oh, right, absolutely. Can we make this a white-list instead then? Just list the verbs we do want (since new ones show up on occasion)